lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -29,33 +29,37 @@
           $mime_split = explode(':', $task->file);
           $task->file = $mime_split[0] . ':' . $mimetype . $mime_split[1];
         }
-        $values = [
+
+        $data = [
           'nid' => $task->campaign_id,
```

Comment:
@angaither i did do some tweaking that should make it better though so I'll push that up as well!